import-export-cli/integration/devFirst_test.go

```diff
@@ -40,6 +43,25 @@ func TestInitializeProject(t *testing.T) {
 
 }
 
+//Initialize an API with --definition flag and import that project
+func TestInitializeAPIWithDefinitionFlag(t *testing.T) {
+	apim := apimClients[0]
+	projectName := "test"
```

Comment:
Shall we have random strings generated for the project names? @uvindra WDYT?

Reply:
+1, the rationale for this is that we can differentiate a given project from another that maybe created by a different test during a test run. This is helpful when we need to debug why a given test is failing.

Reply:
@uvindra Fixed as you suggested